tools/c7n_azure/tests/cassettes/AccessControlTest.test_find_assignments_by_resources.yaml

```diff
@@ -1,145 +1,154 @@
-86bb-9b4d2bc81dc6\"},\"id\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleAssignments/6511a33c-88c5-4dde-ba09-d9c18590ae08\",\"type\":\"Microsoft.Authorization/roleAssignments\",\"name\":\"6511a33c-88c5-4dde-ba09-d9c18590ae08\"},{\"properties\":{\"roleDefinitionId\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c\",\"principalId\":\"f9773e4e-8746-4fff-bada-5d7ecd7fd19e\",\"principalType\":\"ServicePrincipal\",\"scope\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e\",\"createdOn\":\"2018-09-06T23:38:00.9011024Z\",\"updatedOn\":\"2018-09-06T23:38:00.9011024Z\",\"createdBy\":\"139efc82-0e5b-4998-b30a-ce40b9b942db\",\"updatedBy\":\"139efc82-0e5b-4998-b30a-ce40b9b942db\"},\"id\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleAssignments/eb03699e-e93c-433b-b3db-7774769a1d96\",\"type\":\"Microsoft.Authorization/roleAssignments\",\"name\":\"eb03699e-e93c-433b-b3db-7774769a1d96\"},{\"properties\":{\"roleDefinitionId\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c\",\"principalId\":\"fca803fe-453f-4590-8c2b-70ce928ccd4c\",\"principalType\":\"ServicePrincipal\",\"scope\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e\",\"createdOn\":\"2018-05-04T00:08:18.8674444Z\",\"updatedOn\":\"2018-05-04T00:08:18.8674444Z\",\"createdBy\":\"327c0593-bc1a-430c-a8c0-41d3070d389f\",\"updatedBy\":\"327c0593-bc1a-430c-a8c0-41d3070d389f\"},\"id\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleAssignments/ef89f03a-9931-4ce8-aa44-a3a3a74580c7\",\"type\":\"Microsoft.Authorization/roleAssignments\",\"name\":\"ef89f03a-9931-4ce8-aa44-a3a3a74580c7\"},{\"properties\":{\"roleDefinitionId\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleDefinitions/ba92f5b4-2d11-453d-a403-e96b0029c9fe\",\"principalId\":\"3857c640-b81d-4a7b-b006-ca39332fddb2\",\"principalType\":\"User\",\"scope\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e\",\"createdOn\":\"2019-04-25T19:10:25.8774399Z\",\"updatedOn\":\"2019-04-25T19:10:25.8774399Z\",\"createdBy\":\"3857c640-b81d-4a7b-b006-ca39332fddb2\",\"updatedBy\":\"3857c640-b81d-4a7b-b006-ca39332fddb2\"},\"id\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleAssignments/5ae13fc8-0288-40d5-9b1e-35f6ba88c5ec\",\"type\":\"Microsoft.Authorization/roleAssignments\",\"name\":\"5ae13fc8-0288-40d5-9b1e-35f6ba88c5ec\"},{\"properties\":{\"roleDefinitionId\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/providers/Microsoft.Authorization/roleDefinitions/ba92f5b4-2d11-453d-a403-e96b0029c9fe\",\"principalId\":\"579bcd15-51fb-4085-9d23-c6e15a734175\",\"principalType\":\"ServicePrincipal\",\"scope\":\"/subscriptions/aa98974b-5d2a-4d98-a78a-38
```

Comment:
its a little unclear why this is an embedded json string for this recording instead of merged into the top level document as a data structure.